lib/memery.rb

```diff
@@ -49,11 +49,12 @@ def define_memoized_method!(method_name, condition: nil)
           key = [method_name, mod_id].join("_").to_sym
           store = @_memery_memoized_values[key] ||= {}
 
-          if store.key?(args)
-            store[args]
-          else
-            store[args] = super(*args)
+          if store.key?(args) &&
+              (ttl.nil? || Time.now <= store[args][:time] + ttl)
```

Comment:
It's much better to use [monotonic clock](https://blog.dnsimple.com/2018/03/elapsed-time-with-ruby-the-right-way/) in such cases.

Reply:
Thank you for information and suggestion. I've done it.

Please, pay attention: for RuboCop (method length) I've take out `Process.clock_gettime(Process::CLOCK_MONOTONIC)` into private instance method `monotonic_clock`.